### PR TITLE
Lok 21 added .travis.yml configuration for _advanced linting_

### PR DESCRIPTION
**Pull request by matthewpdias**
I updated the .travis.yml file to lint /deploy and /resources directories as well as to continue to lint the Dockerfile. To do this I used yamllint which is installed via pip, thus the language for the travis config is now set to python

### CODE REVIEW

**Comment by ebracho** [comment]
Nice work, I think this is exactly what we were going for. [Build](https://travis-ci.org/Liatrio-LOK/lok-jenkins/jobs/285850747) looks like its running as intended.

**Review by ebracho — APPROVED** [review]
LGTM. Recommend getting a review from @JGuzman4 before merging.

**Review by JGuzman4 — APPROVED** [review]
LGTM

**Pull request merged**
